README.md

```diff
@@ -1,34 +1,56 @@
 # cltdevs meetup manifesto
 
 ## Mission statement
-We grow Charlotte's tech community by bringing people together.
+We are committed to the Charlotte tech community's growth and sustainability by fostering relationships between individuals and organizations, cultivating individual talents and organizations, and discover and build the missing elements of a thriving tech community.
```

Comment:
Very protective of the characters limit on this. I don't think people will remember anything if it's longer than just a few words.